tests/test_units.py

```diff
@@ -46,6 +47,22 @@ def test_convert_unit_with_pint(test_df, current, to):
     assert_converted_units(df, current, to, exp)
 
 
+def test_convert_eqiv_units(test_df):
```

Comment:
Should be enough to just add a line to the parametrisation of `test_convert_gwp()`?

Reply:
Can do, but I don't really trust this as about a quarter of these tests are broken on my machine, I don't understand why.

Reply:
did you install the latest version of `iam_units` from GitHub?

Reply:
nope, turns out it had moved on